integration_tests/mockApis/curiousApi.ts

```diff
@@ -138,13 +138,32 @@ const stubLearnerProfile = (prisonNumber = 'G6115VJ'): SuperAgentRequest =>
     },
   })
 
+const stubLearnerProfile401Error = (prisonNumber = 'G6115VJ'): SuperAgentRequest =>
```

Comment:
We needed an error stub for getting the curious learner profile, in order to display the 'curious is down, please try later' type message